starters/default/src/pages/using-typescript.tsx

```diff
@@ -14,18 +14,21 @@ type DataProps = {
 const UsingTypescript: React.FC<PageProps<DataProps>> = ({ data, path }) => (
   <Layout>
     <Seo title="Using TypeScript" />
-    <h1>Gatsby supports TypeScript by default!</h1>
+    <h1>
+      Gatsby supports <b>TypeScript by default</b>
+    </h1>
     <p>
-      This means that you can create and write <em>.ts/.tsx</em> files for your
-      pages, components etc. Please note that the <em>gatsby-*.js</em> files
-      (like gatsby-node.js) currently don't support TypeScript yet.
+      This means that you can create and write <code>.ts/.tsx</code> files for
+      your pages, components etc. Please note that the <code>gatsby-*.js</code>{" "}
```

Comment:
We can remove this note about gatsby-*.js files now 🎉

Reply:
Thank you, Lennart! 🤗
I tweaked the copy, LMK what you think – too much?